config/settings.py

```diff
@@ -25,6 +26,17 @@
     ".redhat.com",
 ]
 
+OSIDB_MAILING_LIST = os.getenv("OSIDB_MAILING_LIST")
```

Comment:
Nitpick: You might consider using the `osidb.helpers.get_env` helper function so we do have unified approach.